### ISSUE TITLE
are you still maintaning this? if not better just tell people than leave it dormant

### ISSUE THREAD

**Issue by Harrysdog** [MadeBaruna/pom-moe#44]
mostly using the site for timeline, if you have stop maintaning this, please inform

**Comment by JailSeed** [comment]
@MadeBaruna ?

**Comment by scampadversary** [comment]
Yes, please let us know your status, @MadeBaruna, whether you've dropped the site or have gone on hiatus for whatever reason. If you officially announce you're going to end updates, then someone might be more inclined to pick up where you left off rather than wait to see if you have it covered after all. I really appreciate the resource, so I personally hope you come back to it soon! Thank you.

**Comment by daniels1989** [comment]
Safe to say this is abandoned. They're not on hiatus as they're still updating paimon.moe.

**Comment by Idyvist** [comment]
I made a Tampermonkey script to fix most of the warp tracker issues:

https://greasyfork.org/en/scripts/504585-pom-moe-updated/code

Unfortunately I can't really do much about the achievements tracker or the timeline
